Formula/i/imposm3.rb

```diff
@@ -61,8 +61,8 @@ def install
 
     assert_match version.to_s, shell_output("#{bin}/imposm version").chomp
 
-    system "osmium", "cat", testpath/"sample.osm.xml", "-o", "sample.osm.pbf"
-    system "imposm", "import", "-read", testpath/"sample.osm.pbf", "-mapping", testpath/"mapping.yml",
+    system "#{bin}/osmium", "cat", testpath/"sample.osm.xml", "-o", "sample.osm.pbf"
+    system "#{bin}/imposm", "import", "-read", testpath/"sample.osm.pbf", "-mapping", testpath/"mapping.yml",
```

Comment:
actually it would be better to do like this
```suggestion
    system bin/"osmium", "cat", testpath/"sample.osm.xml", "-o", "sample.osm.pbf"
    system bin/"imposm", "import", "-read", testpath/"sample.osm.pbf", "-mapping", testpath/"mapping.yml",
```